legal-api/src/legal_api/services/filings/validations/continuation_in.py

```diff
@@ -171,4 +173,7 @@ def validate_business_in_colin(filing_json: dict, filing_type: str) -> list:
         elif legal_name != response_json['business']['legalName']:
             msg.append({'error': 'Legal name does not match with company legal name from Colin.',
                         'path': business_legal_name_path})
+        elif founding_date != response_json['business']['foundingDate']:
```

Comment:
i dont think `response_json['business']['foundingDate']` is in 'YYYY-MM-DD` format. please verify

Reply:
business.foundingDate" is in "2008-11-28T08:00:00.000+00:00" format.
![image](https://github.com/user-attachments/assets/546734f5-89e1-49d6-a261-43c4f3b99a26)

I think response_json['business']['foundingDate'] format is the same? (like "1983-09-27T00:00:00+00:00")